packages/odyssey-design-tokens/src/font.json

```diff
@@ -10,15 +10,31 @@
         "value": "'Inconsolata', 'SFMono-Regular', 'Consolas', 'Liberation Mono', 'Menlo', 'Courier', monospace"
       }
     },
+    "scale": {
+      "0": { "value": "0.857rem" },
+      "1": { "value": "1rem" },
+      "2": { "value": "1.143rem" },
+      "3": { "value": "1.286rem" },
+      "4": { "value": "1.429rem" },
+      "5": { "value": "1.571rem" },
+      "6": { "value": "1.786rem" },
+      "7": { "value": "2rem" },
+      "8": { "value": "2.286rem" },
+      "9": { "value": "2.571rem" },
+      "10": { "value": "2.857rem" },
+      "11": { "value": "3.214rem" },
+      "12": { "value": "3.643rem" }
+    },
     "size": {
-      "body": { "value": "1rem" },
-      "caption": { "value": "0.871rem" },
-      "heading6": { "value": "1rem" },
-      "heading5": { "value": "1.149rem" },
-      "heading4": { "value": "1.32rem" },
-      "heading3": { "value": "1.516rem" },
-      "heading2": { "value": "1.741rem" },
-      "heading1": { "value": "2rem" }
+      "base": { "value": "14px" },
```

Comment:
this is a good example of a token we should filter out! We never want someone assigning a static 14px value from a token accidentally!

Reply:
We need a tokenized way to set the rem base. It's related to https://github.com/okta/odyssey/pull/1339/files#r817116424

Without a rem-base token, we don't have a way to share the standardized font scale.

Reply:
@edburyenegren-okta Ah I see thank you for additional context, two things:

1) In my experience setting pixel value font-sizes for html is an accessibility violation. It prevents user browser preferences for font size from taking effect. Percentages are OK because they are based on the user preferences. I don't have a specification to reference here this morning but the figma article I linked in our chat previously calls out this same issue.

2) We've talked about this previously in slack and office hours, and never had the opportunity to properly wrap up this conversation but we can also avoid the requirement for downstream teams having to set a font-size for `html` entirely. Instead, they can use our theming system to change font sizes. This is a good outcome because there are fewer externalized dependencies to using Odyssey correctly and it makes future upgrades easier for teams as well. I like this approach better than asking all downstream teams to add an html font-size and make sure they keep it updated alongside any REM unit changes we make into the future - that type of coordination prevents our UI from scaling and being easy to maintain.

Reply:
"...asking all downstream teams to add an html font-size"
This is already the approach that our most used products are taking. This includes Admin, End User, SIW, DevX, and Okta.com. There is no additional lift it teams already use this approach. If however we set our internal base to `0.876rem` instead of `1rem` now those teams have to do the extra work of running a theme to get back to the font size they already set. And someone has to do the work of creating and maintaining those themes.

"...and make sure they keep it updated alongside any REM unit changes we make into the future..." 
All of our design decisions will always be in scale so this isn't a concern. They set the font size they need where they need it and all of our UI scales to meet them.

Rems as a unit scale from the root element, the responsibility for setting the base font size on the page comes from whoever controls the root element. That is the product team not us.

Reply:
As far pixels vs percent, I think this is an area where docs will help. Our token can become a percent if we have some documentation around it saying that it will translate to `14px`. Or out token can remain a pixel value if we have some documentation and a utility for assigning that back to a percent. 

```
html {
  font-size: calc((var(--base) / 16) * 100%);
}
```

Reply:
@andrewberg-okta thanks for response - do we know how many of those teams Admin, End User, SIW, DevX, need to size our font to something other than our default font size? I am recalling some discussion recently about some team needing 16px?

My thought is that if a wants to use a non default font size then the responsibility to build and maintain a theme is on them yes! It should be a very small lift, perhaps only slightly larger than writing `html { font-size: 0.875%; }`  😅 .

When someone does not need to change our defaults**, they have nothing to do.** No setup required in the surrounding browser frame at all to use our components and have them render at the right font size default!

If we back up to the code here `"base": { "value": "14px" }` I think we should reconsider! Exposing this concept as a token, even if it is a percentage! doesn't make a ton of sense for the primary consumers of this API who will be developers building theme-able components. Yes, a team that is using CSS-in-JS can write global styles using it, but so far that is only IGA, and as I mention above, even this can go away if we change the setup required for non default font-sizes.

Reply:
> It should be a very small lift, perhaps only slightly larger than writing html { font-size: 0.875%; }.

To get us aligned on the lift, I want to confirm the non-HTML base proposal. As I understand it, any team wanting to theme away from our current 14px base would need to manually calculate the equivalent proportions and then update:

```
// font.json

    "scale": {
      "0": { "value": "0.857rem" },
      "1": { "value": "1rem" },
      "2": { "value": "1.143rem" },
      "3": { "value": "1.286rem" },
      "4": { "value": "1.429rem" },
      "5": { "value": "1.571rem" },
      "6": { "value": "1.786rem" },
      "7": { "value": "2rem" },
      "8": { "value": "2.286rem" },
      "9": { "value": "2.571rem" },
      "10": { "value": "2.857rem" },
      "11": { "value": "3.214rem" },
      "12": { "value": "3.643rem" }
    },
```

as well as

```
  "space": {
    "scale": {
      "0": { "value": "0.28571429rem" },
      "1": { "value": "0.57142857rem" },
      "2": { "value": "0.85714286rem" },
      "3": { "value": "1.14285714rem" },
      "4": { "value": "1.71428571rem" },
      "5": { "value": "2.28571429rem" },
      "6": { "value": "2.85714286rem" },
      "7": { "value": "3.42857143rem" },
      "8": { "value": "4rem" }
    }
  }
```

That alone is a much larger lift than setting a `font-size` on `html` -- something they will likely do already to accommodate any non-Odyssey UI on the page. We also don't have tokens for every `rem` value in the system (yet), so those would need to be adjusted manually.

It also opens those teams up to breaking design changes if Odyssey (a) adds new tokens (b) changes our desired proportions. Teams would then need to recalculate and update their theme, rather than getting those changes for free.

Overall, it feels like a much larger amount of maintenance than utilizing the standard `rem` behavior.

Reply:
> Exposing this concept as a token, even if it is a percentage doesn't make a ton of sense for the primary consumers of this API who will be developers building theme-able components.

This token should only be utilized at the App/html/body level. Components should only ever be using proportional units to ensure we can drop them into any app and utilize that product's base size.

SIW, for example, could surface this token to their theming interface to allow customers to scale their UI without any extra calculation. Internal customers who aren't only using Odyssey would use it to ensure their non-Odyssey UI matches our standard (e.g. Admin).